e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -38,10 +35,38 @@ Scenario: Re-enrolling an agent
   When the agent is re-enrolled on the host
     And the "agent" process is "started" on the host
   Then the agent is listed in Fleet as online
-    And there is data in the index
+    And there is new data in the index from agent
 
 @revoke-token
 Scenario: Revoking the enrollment token for an agent
   Given an agent is deployed to Fleet
+    And the agent is un-enrolled
+    And the "agent" process is "stopped" on the host
   When the enrollment token is revoked
-  Then an attempt to enroll a new agent fails
+  Then an attempt to enroll an agent with the old token fails
+
+@package-added-to-default-config
+Scenario: Execute packages api calls
+  Given an agent is deployed to Fleet
+    And the package list api returns successfully
+  When the "Cisco" latest package version is installed successfull
+    And a "Cisco" package datasource is added to the 'default' configuration
+  Then the "default" configuration shows the "Cisco" datasource added
+
+@new-agent-configuration
+Scenario: Assign an Agent to a new configuration
+  Given an agent is deployed to Fleet
+    And the agent is listed in Fleet as online
+  When a new configuration named "Test Fleet" is created
+    And the Agent is assigned to the configuration "Test Fleet"
+  Then a new enrollment token is created
+    And there is new data in the index from agent
+
+@new-configuration-new-package
+Scenario: Add a new config and a new package and assign an agent
+  Given an agent is deployed to Fleet
+  When a new configuration named "Test - custom logs" is created
+    And the "custom logs" package datasource is added to the "Test - custom logs" configuration
+    And the Agent is assigned to the configuration "Test - custom logs"
+    And the "Test - custom logs" configuration shows the "custom logs" datasource added
+  Then there is new data in the index from agent from "custom logs" stream
```

Comment:
Wdyt about adding this three scenarios once we complete the above ones?

Reply:
i'd like to hustle thru these as well, they are pertinent and blocking us from expanding out packages testing here.  But if we want to put them to a separate update that's fine.

Reply:
Sure thing! Let me complete the existing ones and I'lll add this. Let's keep this open until then. We are on the right (and fast) track!